kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/manager/LibraryDialogController.java

```diff
@@ -61,9 +61,11 @@
 import com.oracle.javafx.scenebuilder.kit.preferences.MavenPreferences;
 import com.oracle.javafx.scenebuilder.kit.preferences.PreferencesControllerBase;
 import com.oracle.javafx.scenebuilder.kit.preferences.PreferencesRecordArtifact;
+import com.oracle.javafx.scenebuilder.kit.util.ReturningRunnable;
```

Comment:
Please change line 2 from `* Copyright (c) 2016, 2021, Gluon and/or its affiliates.` into `* Copyright (c) 2016, 2022, Gluon and/or its affiliates.`.